index.html

```diff
@@ -15,21 +15,184 @@
           <img src="icons/menu-icon.svg" alt="menu-icon" class="" />
         </button>
+        <ul class="menu">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
       </nav>
     </header>
-    <section class="headline-wrapper">
-      <div class="headline-container">
-        <h1 class="headline-name">I’m Birhanu Glad to see you!</h1>
-        <p class="about-myself">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
+    <section class="headline-container">
+      <h1 class="headline-name">I’m Birhanu Glad to see you!</h1>
+      <p class="about-myself">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social-media">
+        <h2 class="lets-connect">LET’S CONNECT</h2>
+        <ul class="social-icon">
+          <li>
+            <a href="#"> <img src="icons/twitter.svg" /> </a>
+          </li>
+          <li>
+            <a href="#"> <img src="icons/Linkedin icon.svg" /> </a>
+          </li>
+          <li>
+            <a href="#"> <img src="icons/m.svg" /> </a>
+          </li>
+          <li>
+            <a href="#"> <img src="icons/github.svg" /> </a>
+          </li>
+          <li>
+            <a href="#"> <img src="icons/peace.svg" /> </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="works" id="portfolio">
+      <div class="card">
+        <div class="portfolio-snapshot">
+          <img
+            src="assets/images/snapshoot-portfolio-1.png"
+            alt="portfolio-1"
+          />
+        </div>
+        <div class="left-block">
+          <div class="portfolio-info">
+            <h1 class="portfolio-title">TONIC</h1>
+            <ul class="client-role-year">
+              <li class="client">CANOPY</li>
+              <li class="role">Back End Dev</li>
+              <li class="year">2015</li>
+            </ul>
+          </div>
+          <p class="portfolio-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div>
+            <ul class="tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+          </div>
+          <div class="actions">
+            <button class="action action-text btn">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="card">
+        <div class="portfolio-snapshot">
```

Comment:
@Lizdev-05 
What about this one? 
![image](https://user-images.githubusercontent.com/24788565/199819370-5d4a57a4-7c48-4d7d-b4b1-ba9be33e141e.png)

Sorry, I am asking because I have limited number requests pre project! 🥺